lib/susedistribution.pm

```diff
@@ -644,6 +644,14 @@ sub activate_console {
             # case the system is still booting (https://bugzilla.novell.com/show_bug.cgi?id=895602)
             # or when using remote consoles which can take some seconds, e.g.
             # just after ssh login
+            my $retry_num = 0;
+            # retry send ctrl-alt-f key since system not response sometimes
+            while (!(check_screen \@tags, 60) && ($retry_num < 3) && !(check_var('VIRSH_VMM_FAMILY', 'hyperv'))) {
```

Comment:
with this we would check the screen plus additionally check the screen again further down on `assert_screen` on every single loggin in `select_console` call in all tests that do not show any problem. This might be only some seconds but still wasteful. I hope you can find a better approach